tests/common.rs

```diff
@@ -0,0 +1,46 @@
+#[allow(dead_code)]
```

Comment:
How is this dead code? Wouldn't it be preferable to have it gated by the cfg / cfg and split into modules?

Reply:
It's not dead code but rustc thinks it is for some reason.  I'm not sure why, because it really clearly isn't.

Reply:
If you use `common.rs` in multiple tests, each of these will, be compiled as an individual binaries, so each would have to use all functions to not give warnings.